lib/matplotlib/__init__.py

```diff
@@ -207,14 +207,35 @@ def _check_versions():
     sys.argv = ['modpython']
 
 
-_verbose_msg = """\
-matplotlib.verbose is deprecated;
-Command line argument --verbose-LEVEL is deprecated.
-This functionality is now provided by the standard
-python logging library.  To get more (or less) logging output:
-    import logging
-    logger = logging.getLogger('matplotlib')
-    logger.set_level(logging.INFO)"""
+def set_loglevel(level='info'):
+    """
+    Shortcut to set the logging level of the logging module.
+
+    Parameters
+    ----------
+    level : str or bool
```

Comment:
IMHO there is no natural mapping from bool to log levels. I would discard the bool option.
`set_loglevel(True)` is less clear and only slightly less to type compared to `set_loglevel('info')`.